src/components/landing-page/landing-page.controller.ts

```diff
@@ -2,10 +2,39 @@ import * as angular from 'angular';
 
 
 export class LandingPageController implements angular.IController {
+  static $inject = ['$scope', '$timeout'];
 
   public ctrl: any = this;
+  private $scope: any;
+  private $timeout: any;
+
+  constructor ($scope: any, $timeout: any) {
+    this.$scope = $scope;
+    this.$timeout = $timeout;
+  }
 
   public $onInit() {
     this.ctrl.searchText = '';
   }
+
+  public $postLink() {
+    this.$scope.$watch(() => {
+      return this.ctrl.alerts;
+    }, () => {
+      _.forEach(this.ctrl.alerts, (alert: any) => {
```

Comment:
This code looks fine, but I'm worried it's going to be copied and pasted everywhere we want to use toast notifications. I think this would make sense as a service that shows and hides toast notifications, and there is a property in the alert object saying whether it should be automatically hidden after a delay.